### PR TITLE
when changing value for --rules.max-concurrent-evals, –query.max-concurrency this is also need to be change.

### PR DESCRIPTION
**Pull request by kushalShukla-web**

### CODE REVIEW

**Comment by kushalShukla-web** [comment]
fixes https://github.com/prometheus/prometheus/issues/13931

File: cmd/prometheus/main.go [hunk 1]
```diff
@@ -417,7 +417,7 @@ func main() {
 	serverOnlyFlag(a, "rules.alert.resend-delay", "Minimum amount of time to wait before resending an alert to Alertmanager.").
 		Default("1m").SetValue(&cfg.resendDelay)
 
-	serverOnlyFlag(a, "rules.max-concurrent-evals", "Global concurrency limit for independent rules that can run concurrently.").
+	serverOnlyFlag(a, "rules.max-concurrent-evals", "Global concurrency limit for independent rules that can run concurrently.When changing the value for rules.max-concurrent-evals please adjust the value for query.max-concurrency as the rules manager uses the same query engine than the API:").
```
Comment (machine424) [inline cmd/prometheus/main.go:420]:
```suggestion
	serverOnlyFlag(a, "rules.max-concurrent-evals", "Global concurrency limit for independent rules that can run concurrently. When set, \"query.max-concurrency\" may need to be adjusted accordingly.").
```

A small hint will be sufficient I think.

**Review by machine424 — CHANGES_REQUESTED** [review]
thanks!

File: cmd/prometheus/main.go (hunk 1 above)
Comment (kushalShukla-web) [reply cmd/prometheus/main.go:420]: yeah you are right @machine424

**Comment by kushalShukla-web** [comment]
Screenshot from 2024-04-28 06-02-49

**Comment by kushalShukla-web** [comment]
hey @machine424 i think its failing because of some error in tsdb part . can we still merge this PR

**Comment by machine424** [comment]
Actually, the [TestDocumentation](https://github.com/prometheus/prometheus/actions/runs/8866145272/job/24343217892?pr=13965#step:6:346) test is failing.
Did you run `make cli-documentation`?

**Comment by ArthurSens** [comment]
Could you also make sure your PR description is correctly linked with the related issue?

https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

**Comment by kushalShukla-web** [comment]
sure i will do it .

**Comment by kushalShukla-web** [comment]
Oh man I was so busy with exams , I will do this

**Comment by kushalShukla-web** [comment]
i dont know why these changes also get pushed , i stash these changes but still got pushed . will create a new pr .

**Pull request closed without merging**